src/corva/__init__.py

```diff
@@ -15,7 +19,9 @@
     'Api',
     'Cache',
     'Logger',
-    'ScheduledEvent',
```

Comment:
This will be not backward compatible, we will release version 2.0.0, is it okay @jordanambra @ville ?

Reply:
Yes, release it first as beta/rc for QA testing

Reply:
I mean we have 2 options:

1. 
    * Break backward compatibility by totally deleting `ScheduledEvent`
    * Let users know about breaking change by releasing next major version of the library - v2.0.0
 
2.
    * Save backward compatibility by doing following in \_\_init\_\_.py
```python
ScheduledEvent = ScheduledDataTimeEvent


def __getattr__(name):
    if name == "ScheduledEvent":
        warnings.warn(
            "corva.ScheduledEvent usage is deprecated and will be deleted in v2.0.0 use corva.ScheduledDataTimeEvent instead"
        )
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Which one is preffered?